graphql-ballerina/modules/parser/parser.bal

```diff
@@ -127,15 +128,17 @@ public class Parser {
                     Selection selection = {
                         name: name,
                         isFragment: true,
-                        location: location
+                        location: location,
+                        ellipsisLocation: ellipsisLocation
```

Comment:
We should not special case this. The `location` value of the selection should be the `ellipsisLocation` value when the selection is a fragment. Adding another field is not ideal. 

Is there a special reason to add this as a separate field?